app/models/customerInformation/CustomerInformation.scala

```diff
@@ -57,6 +64,8 @@ object CustomerInformation {
   private val changeIndicatorsPath = JsPath \ "changeIndicators"
   private val nameIsReadOnlyPath = JsPath \ "customerDetails" \ "nameIsReadOnly"
   private val partyTypePath = JsPath \ "partyType"
+  private val isInsolventPath = JsPath \ "isInsolvent"
```

Comment:
The two new paths you've added are incorrect. They need to have `"customerDetails"` in the path (in the same way that the fields such as "firstName" and "lastName" do above).

This may have been different on another service (submit?) because it calls a slightly different endpoint which returns a different JSON structure.